pywbem_mock/_baseprovider.py

```diff
@@ -493,3 +493,42 @@ def _get_bare_instance(instance_name, instance_store, copy=None):
             return instance_store.get(instance_name, copy=copy)
 
         return None
+
+    def is_subclass(self, subclass, superclass, class_store):
+        """
+        Return boolean indicating whether subclass is a (direct or indirect)
+        subclass of superclass.
+
+        This test is done by starting at the subclass and walking its
+        superclasses up to the root, so it is not as expensive as determining
+        all subclasses of a given class.
+
+        Parameters:
+
+          subclass (:term:`string`): Class name of subclass.
```

Comment:
Actually why  not use class in instead of  subclass.  It is looking for the suberclass of a class.

Reply:
I know, but it is a name conflict with the 'class' keyword, so we woud need to use `class_` or `klass`.
I'll use `klass`.

Reply:
DONE